www/static/css-src/main.scss

```diff
@@ -194,6 +194,23 @@ code {
     color: #000000;
 }
 
+/* algolia search formatting */
+.algolia-docsearch-suggestion--category-header, .algolia-docsearch-suggestion {
+    font-family: "Raleway", Helvetica, Arial, sans-serif;
+}
```

Comment:
Is the font being overridden in the Algolia CSS file? If not, the existing font rules for `body` should cover fonts for this widget.